content/en/docs/Writing policies/match-exclude.md

```diff
@@ -8,12 +8,24 @@ The `match` and `exclude` filters control which resources policies are applied t
 
 The `match` and `exclude` clauses have the same structure and can each contain the following elements:
 
+* `any`: specify [resource filters](#resource-filters) on which Kyverno will perform the logical **OR** operation while choosing resources
```

Comment:
If the limitation exists as discussed yesterday whereby `any` and `all` are mutually exclusive with only one of either possible then this needs to be stated here.